Sources/API/Debug.swift

```diff
@@ -19,119 +19,147 @@ public class Debug {
 // MARK: Standard API
 
 extension Debug {
-    /// Output `target` to console.
+    /// Output `targets` to console.
     /// - Parameters:
-    ///   - target: target
+    ///   - targets: targets
+    ///   - separator: A string to print between each item.
     public static func print(
-        _ target: Any
+        _ targets: Any...,
+        separator: String = " "
```

Comment:
This default value `" "` is similar to standard library.
https://developer.apple.com/documentation/swift/1541053-print